tedana/reporting/static_figures.py

```diff
@@ -798,3 +800,102 @@ def plot_adaptive_mask(
     )
     adaptive_mask_plot = f"{io_generator.prefix}adaptive_mask.svg"
     fig.savefig(os.path.join(io_generator.out_dir, "figures", adaptive_mask_plot))
+
+
+def clustering_results(cluster_labels, similarity_t_sne, io_generator):
+    """Plot the clustering results of robustica.
+
+    Parameters
+    ----------
+    cluster_labels : (n_pca_components x n_robust_runs,) : numpy.ndarray
+        A one dimensional array that has the cluster label of each run.
+    similarity_t_sne : (n_pca_components x n_robust_runs,2) : numpy.ndarray
+        An array containing the coordinates of projected data.
+    io_generator : object
+        An object containing all the information needed to generate the output.
+    """
+    title = "2D projection of clustered ICA runs using TSNE"
+    line_width = 0.6
+    marker_size = 3
+    convex_hull_color = "b--"
+    point_color = "k"
+    dilation_factor = 1.5
```

Comment:
These (and potentially other plotting variables later in the function) could be defined as constants at the head of the file, to distinguish them from source code.